.travis.yml

```diff
@@ -1,13 +1,13 @@
 language: java
 jdk:
-  - openjdk7
-  - oraclejdk7
-  - oraclejdk8
-install: /bin/true
-script: mvn checkstyle:check test package
+- openjdk7
+- oraclejdk7
+- oraclejdk8
```

Comment:
Is there any specific reason why you are abandoning indentation of those itemized lists? All the [examples] (http://docs.travis-ci.com/user/languages/java/#Testing-Against-Multiple-JDKs) I can find in the travis docs show itemized lists with indentation.